osu.Server.BeatmapSubmission/Program.cs

```diff
@@ -60,6 +60,28 @@ public static void Main(string[] args)
                 }
 
                 case "Staging":
+                {
+                    builder.Services.AddSingleton<IBeatmapStorage, LocalBeatmapStorage>();
+                    builder.Services.AddTransient<BeatmapPackagePatcher>();
+                    builder.Services.AddHttpClient();
+                    builder.Services.AddTransient<ILegacyIO, LegacyIO>();
+                    builder.Services.AddTransient<IMirrorService, NoOpMirrorService>();
+
+                    if (AppSettings.SentryDsn == null)
+                    {
+                        throw new InvalidOperationException("SENTRY_DSN environment variable not set. "
+                                                            + "Please set the value of this variable to a valid Sentry DSN to use for logging events.");
+                    }
+
+                    if (AppSettings.DatadogAgentHost == null)
+                    {
+                        throw new InvalidOperationException("DD_AGENT_HOST environment variable not set. "
```

Comment:
We don't use ddog on staging 💸

Reply:
Roger, made optional in https://github.com/ppy/osu-server-beatmap-submission/pull/17/commits/8d398e1714c37a23e6da927ee030169232e98c2c. Note that after that, if the envvar is not set, ddog will not be set up:

https://github.com/ppy/osu-server-beatmap-submission/blob/82190f3130bae07747fe03a2eadf8cafa81fe401/osu.Server.BeatmapSubmission/Program.cs#L116-L128